### PR TITLE
Add Identifier and "contact" to accessibility.digital.gov

### PR DESCRIPTION
**Pull request by nick-mon1**
Added `Identifier` markup to footer. Need to update `uswds` and styles.

### CODE REVIEW

**Comment by ToniBonittoGSA** [comment]
@nick-mon1 it looks a bit off in the preview: https://federalist-ed9ac7ce-8870-4699-95f8-e85f8b3c15b8.app.cloud.gov/preview/gsa/accessibility-for-teams/nl-add-identifier-component/

File: _includes/footer.html [hunk 1]
```diff
@@ -1,18 +1,89 @@
...
-      <a class="footer-button usa-text-small" href="https://github.com/18F/accessibility-playbook/issues/"><i class="fa fa-github" aria-hidden="true"></i>File an issue</a>
-    </div>
+    <b>Contact:</b> Questions or comments? Please file a
+    <a href="https://github.com/18F/accessibility-playbook/issues/" class="text-ink text-bold hover:text-no-underline">GitHub issue</a>
+    or email
+    <a href="mailto:g-accessibility@gsa.gov" class="text-ink text-bold hover:text-no-underline">g-accessibility@gsa.gov</a>.
+  </div>
+</div>
+
+<footer class="site-footer" role="contentinfo">
```
Comment (iamjolly) [inline _includes/footer.html:10]: `role="contentinfo"` is redundant for a standalone `footer` element. It doesn't hurt anything, but it is extra code that only restates the footer's default/inherent role.

**Comment by iamjolly** [comment]
Also, thank you @nick-mon1 for working on this site and updating to USWDS3!

File: _includes/footer.html [hunk 2]
```diff
@@ -1,18 +1,89 @@
...
+  <div class="usa-identifier">
+    <section class="usa-identifier__section usa-identifier__section--masthead" aria-label="Agency identifier">
+      <div class="usa-identifier__container">
+        <div class="usa-identifier__logos">
+          <a href="https://www.gsa.gov" class="usa-identifier__logo">
+            <!-- <img class="usa-identifier__logo-img" src="/img/gsa-logo-w100.png" alt="GSA logo" role="img"> -->
+            <img class="usa-identifier__logo-img" src="{{ site.baseurl }}/assets/img/gsa-logo.svg" alt="GSA logo" role="img">
```
[the remark this answers is not in the document]
Comment (nick-mon1) [reply _includes/footer.html]: role is now removed

File: _includes/footer.html [hunk 3]
```diff
@@ -1,18 +1,89 @@
...
+          </a>
+        </div>
+        <div class="usa-identifier__identity" aria-label="Agency description">
+          <p class="usa-identifier__identity-domain">accessbility.digital.gov</p>
```
[the remark this answers is not in the document]
Comment (nick-mon1) [reply _includes/footer.html]: typo is now fixed

File: _includes/search-form.html [hunk 4]
```diff
@@ -8,4 +8,26 @@
       <span class="usa-sr-only">Search</span>
     </button>
   </div>
-</form>
\ No newline at end of file
+</form> -->
+
+<section aria-label="Small search component">
```
[the remark this answers is not in the document]
Comment (nick-mon1) [reply _includes/search-form.html]: the extra aria-label is now removed

File: _includes/search-form.html [hunk 5]
```diff
@@ -8,4 +8,26 @@
...
+  <form id="search_form" class="usa-search usa-search--small" role="search" accept-charset="UTF-8" action="https://search.usa.gov/search" method="get">
+    <input name="utf8" type="hidden" value="&#x2713;" />
+    <input type="hidden" name="affiliate" id="affiliate" value="a11y4teams" />
+    <label class="usa-sr-only" for="search-field-en-small" for="{{ include.id }}">Search</label>
```
[the remark this answers is not in the document]
Comment (nick-mon1) [reply _includes/search-form.html]: thanks for the catch, this is now in sync

File: _includes/header.html [hunk 6]
```diff
@@ -5,37 +5,84 @@
...
+            Here’s how you know
+          </p>
         </div>
-        <button class="usa-accordion__button usa-banner__button"
-          aria-expanded="false" aria-controls="gov-banner-demo">
+        <button
+          class="usa-accordion__button usa-banner__button"
+          aria-expanded="false"
+          aria-controls="gov-banner-default-default"
+        >
```
[the remark this answers is not in the document]
Comment (nick-mon1) [reply _includes/header.html:26]: Updated to use `uswds 3.0` which fixes this with the new component code

File: _includes/footer.html (hunk 1 above)
Comment (nick-mon1) [reply _includes/footer.html]: this is now removed

File: _includes/footer.html [hunk 7]
```diff
@@ -1,18 +1,89 @@
...
-    <div class="footer-text">
-      <p class="usa-text-small">A project of GSA’s <a href="https://www.gsa.gov/about-us/organization/federal-acquisition-service/technology-transformation-services">Technology Transformation Services</a>, maintained by the <a href="https://accessibility.18f.gov/">Accessibility Guild</a>.</p>
-      <p class="usa-text-small">This website is hosted on <a href="https://federalist.18f.gov/">Federalist</a>.</p>
-    </div>
-    <div class="footer-buttons">
-      <a class="footer-button usa-text-small" href="https://github.com/18F/accessibility-playbook/edit/main/{{ page.path }}"><i class="fa fa-pencil" aria-hidden="true"></i>Edit this page</a>
-      <a class="footer-button usa-text-small" href="https://github.com/18F/accessibility-playbook/issues/"><i class="fa fa-github" aria-hidden="true"></i>File an issue</a>
-    </div>
+    <b>Contact:</b> Questions or comments? Please file a
+    <a href="https://github.com/18F/accessibility-playbook/issues/" class="text-ink text-bold hover:text-no-underline">GitHub issue</a>
```
[the remark this answers is not in the document]
Comment (nick-mon1) [reply _includes/footer.html]: updated with the new address

**Review by mejiaj — CHANGES_REQUESTED** [review]
Nice cleanup! Added some additional changes that might help even more.

File: .gitignore [hunk 8]
```diff
@@ -4,3 +4,8 @@ _site
 node_modules/
 .jekyll-cache
 material-icons/
+.DS_Store
+
+assets/img/lib/uswds/material-icons-deprecated/
+assets/img/lib/uswds/usa-icons-bg/
+assets/img/lib/uswds/uswds-icons/
```
[the remark this answers is not in the document]
Comment (nick-mon1) [reply .gitignore:11]: I've made a note to follow up on this

File: _includes/head.html [hunk 9]
```diff
@@ -20,11 +20,13 @@
   <meta property="og:url" content="{{ page.url | absolute_url }}">
 
   <link rel="stylesheet" href="{{ "/assets/fonts/font-awesome.min.css" | relative_url }}">
-  <link rel="stylesheet" href="{{ "/assets/stylesheets/site/main.css" | relative_url }}">
+  <link rel="stylesheet" href="{{ "/assets/stylesheets/css/main.css" | relative_url }}">
   <link rel="alternate" type="application/rss+xml" title="{{ site.title | escape }}" href="{{ "/feed.xml" | relative_url }}">
   <link rel="icon" type="image/png" href="{{ "/assets/img/favicon-32x32.png" | relative_url }}" sizes="32x32" />
   <link rel="icon" type="image/png" href="{{ "/assets/img/favicon-16x16.png" | relative_url }}" sizes="16x16" />
 
+  <script defer src="{{ "assets/js/lib/uswds/uswds.min.js" | relative_url }}"></script>
```
[the remark this answers is not in the document]
Comment (nick-mon1) [reply _includes/head.html:28]: Good catch, I've updated to include the init script

File: gulpfile.js [hunk 10]
```diff
@@ -29,19 +31,19 @@ PATHS
...
 // Images destination
 uswds.paths.dist.img = "./assets/img/lib/uswds";
 
 // Fonts destination
-uswds.paths.dist.fonts = "./assets/fonts";
+uswds.paths.dist.fonts = "./assets/fonts/lib/uswds";
 
 // Javascript destination
-uswds.paths.dist.js = "./assets/js";
+uswds.paths.dist.js = "./assets/js/lib/uswds";
```
[the remark this answers is not in the document]
Comment (nick-mon1) [reply gulpfile.js:43]: Yes, they do get generated on my machine

File: package.json [hunk 11]
```diff
@@ -19,7 +19,9 @@
...
   "devDependencies": {
     "@uswds/compile": "^1.0.0-beta.2",
-    "gulp": "^4.0.2",
-    "uswds": "^2.8.1"
+    "gulp": "^4.0.2"
+  },
+  "dependencies": {
+    "@uswds/uswds": "3.0.2"
   }
 }
```
[the remark this answers is not in the document]
Comment (nick-mon1) [reply package.json:27]: `npm start` command is updated with the above changes

**Comment by iamjolly** [comment]
@nick-mon1 - Question for you... do the [README instructions for running the site locally](https://github.com/GSA/accessibility-for-teams#running-the-site) need any updates with this PR? I'm guessing no, but I wanted to ask just in case.

**Comment by nick-mon1** [comment]
@iamjolly Good catch, I've updated the [README.md](https://github.com/GSA/accessibility-for-teams#running-the-site) instructions because they have changed.

**Review by iamjolly — APPROVED** [review]
Thank you @nick-mon1 for all this work!

**Review by iamjolly — APPROVED** [review]

**Pull request merged**